src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1209,6 +1224,22 @@ private void SetEnableHttp2PlusClientCertificate(Uri requestUri, Version request
             }
         }
 
+        private void SetDisableHttp2StreamQueue()
+        {
+            if (_enableMultipleHttp2Connections)
+            {
+                uint optionData = 1;
+                if (Interop.WinHttp.WinHttpSetOption(_sessionHandle, Interop.WinHttp.WINHTTP_OPTION_DISABLE_STREAM_QUEUE, ref optionData))
+                {
+                    if (NetEventSource.IsEnabled) NetEventSource.Info(this, "Multiple HTTP/2 connections enabled.");
+                }
+                else
+                {
+                    if (NetEventSource.IsEnabled) NetEventSource.Info(this, "Multiple HTTP/2 connections cannot be enabled.");
+                }
```

Comment:
Nit: This could be simplified a little, e.g.
```C#
bool success = Interop.WinHttp.WinHttpSetOption(_sessionHandle, Interop.WinHttp.WINHTTP_OPTION_DISABLE_STREAM_QUEUE, ref optionData);
if (NetEventSource.Log.IsEnabled()) NetEventSource.Info(this, success ? "Multiple HTTP/2 connections enabled." : "Multiple HTTP/2 connections cannot be enabled.");
```